Перевод расовых действий

PhantornRU opened the pull request:
- Перевод [РОСКОМНАДЗОР] действий при прописывании команды /suicide для всех игровых рас (да, даже для скелетов и армалисов)
- Перевод фраз и описаний IPC, в том числе смерти.
- Перевод текстовых описаний возгораний плазмамена, а также акцент речи с заменой "с"/"ш" на "шш"
- Перевод текстовых описаний урона тенелинга
- Перевод тектовых описаний слайма и его восстановительных процессов
- Перевод текстовых описаний врина, его укалывание жалом, отрывание антены

PhantornRU committed:
Перевод расовых действий

- Перевод [РОСКОМНАДЗОР] действий при прописывании команды /suicide для всех игровых рас (да, даже для скелетов и армалисов)
- Перевод фраз и описаний IPC, в том числе смерти.
- Перевод текстовых описаний возгораний плазмамена, а также акцент речи с заменой "с"/"ш" на "шш"
- Перевод текстовых описаний урона тенелинга
- Перевод тектовых описаний слайма и его восстановительных процессов
- Перевод текстовых описаний врина, его укалывание жалом, отрывание антены

Ohar commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -61,9 +61,10 @@
-		"is pulling themselves apart!")
+		"теряет ветви!",
+		"вытаскивает из тайника бутыль с гербицидом и делает большой глоток!",
+		"растаскивается на части!",
```
Растаскивается кем?

Ohar commented on code/modules/mob/living/carbon/human/species/drask.dm:
```diff
@@ -27,10 +27,10 @@
-		"is jamming fingers through their big eyes!",
-		"is sucking in warm air!",
-		"is holding their breath!")
+		"нагревается самотрением!",
```
Самотрение лучше заменить на иной словесный оборот
```suggestion
		"трёт себя до возгорания!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -63,8 +63,8 @@
-		"is crumbling into dust!",
-		"is smashing their body apart!")
+		"рассыпается в прах!",
+		"разбивает свое тело на части!")
```
Ё пропустил
```suggestion
		"разбивает своё тело на части!")
```

Ohar commented on code/modules/mob/living/carbon/human/species/kidan.dm:
```diff
@@ -32,29 +32,15 @@
-		"l_foot" = list("path" = /obj/item/organ/external/foot),
-		"r_foot" = list("path" = /obj/item/organ/external/foot/right)
-		)
-
```
А куда делась проверка конечностей?

PhantornRU replied:
проверка конечностей не мой коммит, это кто-то недавно выкатывал его из-за бага с головой Кидана.
В общем конфликт возник. Завтра тогда сравню с ПРом головы киданов

Ohar commented on code/modules/mob/living/carbon/human/species/kidan.dm:
```diff
@@ -32,29 +32,15 @@
-		"is stabbing themselves with their mandibles!",
-		"is holding their breath!")
+		"пытается откусить свои усики!",
+		"вонзает когти в свои глазницы!",
```
Замени «свои» на «себе» в обоих случаях

Ohar commented on code/modules/mob/living/carbon/human/species/kidan.dm:
```diff
@@ -32,29 +32,15 @@
+		"пытается откусить свои усики!",
+		"вонзает когти в свои глазницы!",
+		"сворачивает себе шею!",
+		"трескается экзоскелетом!",
```
```suggestion
		"разбивает свой панцирь",
```

Ohar commented on code/modules/mob/living/carbon/human/species/machine.dm:
```diff
@@ -19,7 +19,7 @@
 	tox_mod = 0
 	clone_mod = 0
-	death_message = "gives a short series of shrill beeps, their chassis shuddering before falling limp, nonfunctional."
+	death_message = "издает серию коротких пронзительных звуков, корпус вздрагивает перед тем как отключиться, больше не работоспособно."
```
```suggestion
	death_message = "издаёт резкие пронзительные звуки и, конвульсивно подёргивая шасси,  окончательно отключается."
```

Ohar commented on code/modules/mob/living/carbon/human/species/machine.dm:
```diff
@@ -132,7 +132,7 @@
 		var/optic_colour = input(H, "Select optic colour", H.m_colours["head"]) as color|null
 		if(H.incapacitated(TRUE, TRUE, TRUE))
-			to_chat(H, "<span class='warning'>You were interrupted while changing the colour of your optics.</span>")
+			to_chat(H, "<span class='warning'>Вас прервали, при смене цвета оптики.</span>")
```
```suggestion
			to_chat(H, "<span class='warning'>Ваша попытка сменить отображаемый цвет была прервана.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/machine.dm:
```diff
@@ -160,7 +160,7 @@
 
 		if(H.incapacitated(TRUE, TRUE, TRUE))
-			to_chat(H, "<span class='warning'>You were interrupted while changing your monitor display.</span>")
+			to_chat(H, "<span class='warning'>Вас прервали, при смене изображения на дисплее.</span>")
```
Аналогично

Ohar commented on code/modules/mob/living/carbon/human/species/plasmaman.dm:
```diff
@@ -24,10 +24,10 @@
-		"shows their true colors, which happens to be the color of plasma!")
+		"сворачивает себе шею!",
+		"впускает себе немного O2!",
+		"осознает экзистенциальную проблему рожденным из плазмы!",
```
```suggestion
		"осознает экзистенциальную проблему быть рождённым из плазмы!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/plasmaman.dm:
```diff
@@ -24,10 +24,10 @@
+		"сворачивает себе шею!",
+		"впускает себе немного O2!",
+		"осознает экзистенциальную проблему рожденным из плазмы!",
+		"показывает свое истинное лицо, которое оказалось цветом плазмы!")
```
```suggestion
		"показывает свою истинную природу, которая оказывается плазмой!")
```

Ohar commented on code/modules/mob/living/carbon/human/species/shadow.dm:
```diff
@@ -22,10 +22,10 @@
-		"is twisting their own neck!",
-		"is staring into the closest light source!")
+		"пытается откусить себе язык!",
+		"выдавливает свои глазницы большими пальцами!",
```
```suggestion
		"выдавливает большими пальцами себе глазницы!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/shadow.dm:
```diff
@@ -40,10 +40,10 @@
 	if(!H.vision_type)
 		H.set_sight(/datum/vision_override/nightvision)
-		to_chat(H, "<span class='notice'>You adjust your vision to pierce the darkness.</span>")
+		to_chat(H, "<span class='notice'>Вы расширяете свой взор, чтобы видеть сквозь тьму.</span>")
```
```suggestion
		to_chat(H, "<span class='notice'>Вы изменяете свой взор, чтобы видеть сквозь тьму.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/shadow.dm:
```diff
@@ -40,10 +40,10 @@
 	else
 		H.set_sight(null)
-		to_chat(H, "<span class='notice'>You adjust your vision to recognize the shadows.</span>")
+		to_chat(H, "<span class='notice'>Вы расширяете свой взор, чтобы распознавать тени.</span>")
```
```suggestion
		to_chat(H, "<span class='notice'>Вы изменяете свой взор, чтобы вновь различать свет и тени.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/shadowling.dm:
```diff
@@ -31,7 +31,7 @@
 			H.take_overall_damage(0, LIGHT_DAMAGE_TAKEN)
 			if(H.stat != DEAD)
-				to_chat(H, "<span class='userdanger'>The light burns you!</span>")//Message spam to say "GET THE FUCK OUT"
+				to_chat(H, "<span class='userdanger'>Свет обжигает тебя!</span>")//Message spam to say "GET THE FUCK OUT"
```
```suggestion
				to_chat(H, "<span class='userdanger'>Свет жжёт вас!</span>")//Message spam to say "GET THE FUCK OUT"
```

Ohar commented on code/modules/mob/living/carbon/human/species/skeleton.dm:
```diff
@@ -33,9 +33,10 @@
-		"is snapping their own bones!",
-		"is collapsing into a pile!",
-		"is twisting their skull off!")
+		"ломает свои кости!",
```
```suggestion
		"ломает себе кости!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/skrell.dm:
```diff
@@ -40,11 +40,11 @@
-		"is twisting their own neck!",
-		"makes like a fish and suffocates!",
-		"is strangling themselves with their own tendrils!")
+		"пытается откусить свой язык!",
```
```suggestion
		"пытается откусить себе язык!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -128,10 +130,10 @@
 	if(S.reagent_skin_coloring)
 		S.reagent_skin_coloring = FALSE
-		to_chat(H, "You adjust your internal chemistry to filter out pigments from things you consume.")
+		to_chat(H, "Вы настраиваете свою внутреннюю химию, чтобы отфильтровывать пигменты из продуктов, которые вы потребляете.")
```
```suggestion
		to_chat(H, "Вы настраиваете свою внутреннюю химию, чтобы отфильтровывать пигменты из употребляемых продуктов.")
```

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -128,10 +130,10 @@
 	else
 		S.reagent_skin_coloring = TRUE
-		to_chat(H, "You adjust your internal chemistry to permit pigments in chemicals you consume to tint you.")
+		to_chat(H, "Вы настраиваете свою внутреннюю химию, чтобы позволить употребленным пигментам в химических веществах, окрасить вас.")
```
```suggestion
		to_chat(H, "Вы настраиваете свою внутреннюю химию, позволяя окрашивать себя пигментами употребляемых веществ.")
```

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -158,30 +160,30 @@
 	var/chosen_limb = missing_limbs[limb_select]
 
-	H.visible_message("<span class='notice'>[H] begins to hold still and concentrate on [H.p_their()] missing [limb_select]...</span>", "<span class='notice'>You begin to focus on regrowing your missing [limb_select]... (This will take [round(SLIMEPERSON_REGROWTHDELAY/10)] seconds, and you must hold still.)</span>")
+	H.visible_message("<span class='notice'>[H] замирает и концентрируется на [H.p_their()] потерянной [limb_select]...</span>", "<span class='notice'>Вы концентрируетесь на отращивании [limb_select]... (Это займет [round(SLIMEPERSON_REGROWTHDELAY/10)] секунд, нужно подождать в спокойствии.)</span>")
```
Сложное место
Придумай как склонять название конечности
Добавь склонение числа секунд функцией `declension_ru`
```suggestion
	H.visible_message("<span class='notice'>[H] замирает и концентрируется на своей потерянной [limb_select]...</span>", "<span class='notice'>Вы концентрируетесь на отращивании [limb_select]... (Это займет [round(SLIMEPERSON_REGROWTHDELAY/10)] секунд, нужно подождать в спокойствии.)</span>")
```

PhantornRU replied:
это цифры, им не нужны склонения
![image](https://user-images.githubusercontent.com/41479614/168453397-dc7fab40-6699-4167-927c-a8e8b9d280ef.png)

PhantornRU replied:
Конечности русифицированы
![image](https://user-images.githubusercontent.com/41479614/168454017-b69f4ce4-fd65-44c0-9a30-23663f95a749.png)
![image](https://user-images.githubusercontent.com/41479614/168454019-f8be693d-f7da-4b27-8902-c472e4d946f9.png)

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -158,30 +160,30 @@
 		var/stored_burn = 0
 		if(istype(O))
-			to_chat(H, "<span class='warning'>You distribute the damaged tissue around your body, out of the way of your new pseudopod!</span>")
+			to_chat(H, "<span class='warning'>Вы распределяете поврежденную ткань по всему телу, освобождая место, для ложноножки!</span>")
```
Запятая лишняя 
```suggestion
			to_chat(H, "<span class='warning'>Вы распределяете поврежденную ткань по всему телу, освобождая место для ложноножки!</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -192,7 +194,7 @@
 		var/obj/item/organ/external/potential_parent = H.bodyparts_by_name[initial(limb_path.parent_organ)]
 		if(!istype(potential_parent))
-			to_chat(H, "<span class='danger'>You've lost the organ that you've been growing your new part on!</span>")
+			to_chat(H, "<span class='danger'>Вы потеряли орган, на котором выращивали вашу новую часть!</span>")
```
```suggestion
			to_chat(H, "<span class='danger'>Вы потеряли орган, на котором выращивали новую конечность!</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -204,9 +206,9 @@
 		H.UpdateDamageIcon()
 		H.adjust_nutrition(-SLIMEPERSON_HUNGERCOST)
-		H.visible_message("<span class='notice'>[H] finishes regrowing [H.p_their()] missing [new_limb]!</span>", "<span class='notice'>You finish regrowing your [limb_select]</span>")
+		H.visible_message("<span class='notice'>[H] завершает отращивание [H.p_their()] потерянной [new_limb]!</span>", "<span class='notice'>Вы завершили отращивание [limb_select]</span>")
```
```suggestion
		H.visible_message("<span class='notice'>[H] отращивает [H.p_their()] потерянную [new_limb]!</span>", "<span class='notice'>Вы отрастили [limb_select]</span>")
```

PhantornRU replied:
Внёс

Ohar commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -204,9 +206,9 @@
+		H.visible_message("<span class='notice'>[H] завершает отращивание [H.p_their()] потерянной [new_limb]!</span>", "<span class='notice'>Вы завершили отращивание [limb_select]</span>")
 	else
-		to_chat(H, "<span class='warning'>You need to hold still in order to regrow a limb!</span>")
+		to_chat(H, "<span class='warning'>Вам нужно подождать в спокойствии чтобы отрастить конечность!</span>")
```
```suggestion
		to_chat(H, "<span class='warning'>Для отращивания конечности вам нужно стоять на месте!</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/tajaran.dm:
```diff
@@ -64,10 +64,10 @@
-		"is twisting their own neck!",
-		"is holding their breath!")
+		"пытается откусить себе язык!",
+		"вонзает когти в глазницы!",
```
```suggestion
		"вонзает когти себе в глазницы!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/unathi.dm:
```diff
@@ -66,10 +66,10 @@
-		"is twisting their own neck!",
-		"is holding their breath!")
+		"пытается откусить себе язык!",
+		"вонзает когти в глазницы!",
```
```suggestion
		"вонзает когти себе в глазницы!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/vox.dm:
```diff
@@ -81,11 +81,11 @@
-		"is twisting their own neck!",
-		"is holding their breath!",
-		"is deeply inhaling oxygen!")
+		"пытается откусить свой язык!",
```
```suggestion
		"пытается откусить себе язык!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/vox.dm:
```diff
@@ -81,11 +81,11 @@
-		"is holding their breath!",
-		"is deeply inhaling oxygen!")
+		"пытается откусить свой язык!",
+		"вонзает когти в глазницы!",
```
```suggestion
		"вонзает когти себе в глазницы!",
```

Ohar commented on code/modules/mob/living/carbon/human/species/vox.dm:
```diff
@@ -215,11 +215,11 @@
-		"is holding their breath!",
-		"is huffing oxygen!")
+		"пытается откусить свой язык!",
+		"вонзает когти в глазницы!",
```
Аналогично

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -79,8 +79,8 @@
 /datum/action/innate/wryn_sting
-	name = "Wryn Sting"
-	desc = "Readies Wryn Sting for stinging."
+	name = "Жало Врина"
```
На русском названия рас не считаются именами собственными и пишутся со строчной буквы
```suggestion
	name = "Жало врина"
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -79,8 +79,8 @@
-	name = "Wryn Sting"
-	desc = "Readies Wryn Sting for stinging."
+	name = "Жало Врина"
+	desc = "Подготовка жала Врина для ужаливания."
```
```suggestion
	desc = "Подготовка жала к ужаливанию."
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -91,13 +91,13 @@
 		return
 	if(user.wear_suit)	//Is your Wryn wearing a Hardsuit or a Laboat that's blocking their Stinger?
-		to_chat(user, "<span class='notice'>You must remove your hardsuit, labcoat, or jacket before using your Wryn stinger.</span>")
+		to_chat(user, "<span class='notice'>Нужно снять верхнюю одежду перед тем как использовать жало Врина.</span>")
```
```suggestion
		to_chat(user, "<span class='notice'>Для использования жала нужно снять верхнюю одежду.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -91,13 +91,13 @@
 		return
 	if(user.getStaminaLoss() >= 50)	//Does your Wryn have enough Stamina to sting?
-		to_chat(user, "<span class='notice'>You feel too tired to use your Wryn Stinger at the moment.</span>")
+		to_chat(user, "<span class='notice'>Вы чувствуете себя слишком усталым чтобы использовать жало Врина.</span>")
```
```suggestion
		to_chat(user, "<span class='notice'>Вы слишком устали для использования жала.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
 	if(!target)		//No one's around!
-		to_chat(user, "<span class='warning'>There's no one around to sting so you retract your stinger.</span>")
-		user.visible_message("<span class='warning'[user] retracts their stinger.</span>")
+		to_chat(user, "<span class='warning'>Вокруг некого ужалить! Жало втягивается обратно.</span>")
```
```suggestion
		to_chat(user, "<span class='warning'>Вокруг некого жалить! Жало втягивается обратно.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
-		to_chat(user, "<span class='warning'>There's no one around to sting so you retract your stinger.</span>")
-		user.visible_message("<span class='warning'[user] retracts their stinger.</span>")
+		to_chat(user, "<span class='warning'>Вокруг некого ужалить! Жало втягивается обратно.</span>")
+		user.visible_message("<span class='warning'[user] втягивает свое жало.</span>")
```
```suggestion
		user.visible_message("<span class='warning'[user] втягивает своё жало.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
 		return
 	else			//Get ready, aim, fire!
-		user.visible_message("<span class='warning'> [user] prepares to use their Wryn stinger!</span>")
+		user.visible_message("<span class='warning'> [user] готовится использовать жало Врина!</span>")
```
```suggestion
		user.visible_message("<span class='warning'> [user] собирается применить жало!</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
 	if(!(target in orange(1, user)))	//Dang, did they get away?
-		to_chat(user, "<span class='warning'>You are no longer adjacent to [target]. You retract your stinger for now.</span>")
-		user.visible_message("<span class='warning'[user] retracts their stinger.</span>")
+		to_chat(user, "<span class='warning'>Вы далеко от [target]. Вы убираете свое жало.</span>")
```
```suggestion
		to_chat(user, "<span class='warning'>Вы слишком далеко от [target]. Жало втягивается.</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
 		var/obj/item/organ/external/organ = target.get_organ(pick("l_leg", "r_leg", "l_foot", "r_foot", "groin"))
-		to_chat(user, "<span class='danger'> You sting [target] in their [organ] with your stinger!</span>")
-		user.visible_message("<span class='danger'>[user] stings [target] in [organ] with their stinger! </span>")
+		to_chat(user, "<span class='danger'> Вы жалите [target] в [organ] своим жалом!</span>")
```
Масло масляное
```suggestion
		to_chat(user, "<span class='danger'> Вы жалите [target] в [organ]!</span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -123,28 +123,28 @@
-		to_chat(user, "<span class='danger'> You sting [target] in their [organ] with your stinger!</span>")
-		user.visible_message("<span class='danger'>[user] stings [target] in [organ] with their stinger! </span>")
+		to_chat(user, "<span class='danger'> Вы жалите [target] в [organ] своим жалом!</span>")
+		user.visible_message("<span class='danger'>[user] жалит [target] в [organ] его жалом! </span>")
```
```suggestion
		user.visible_message("<span class='danger'>[user] жалит [target] в [organ]! </span>")
```

Ohar commented on code/modules/mob/living/carbon/human/species/wryn.dm:
```diff
@@ -163,22 +163,22 @@
-			to_chat(target, "<span class='danger'>You hear a loud crunch as your antennae is ripped off your head by [user].</span>")
-			to_chat(target, "<span class='danger'><B>It's so quiet...</B></span>")
+			to_chat(user, "<span class='notice'>Вы слышите громкий хруст, когда безжалостно отрываете усики [target].</span>")
+			to_chat(target, "<span class='danger'>Вы слышите невыносимый громкий хруст, когда ваши усики оторваны [user].</span>")
```
```suggestion
			to_chat(target, "<span class='danger'>Вы слышите невыносимый хруст, когда [user] вырывает усики из вашей головы.</span>")
```

Ohar reviewed (changes requested):
Общее замечание: почти нигде не учтён пол персонажей (не забывай про множественное число).
Строки с названиями органов нужно сильно перестроить или родить способ их склонять. «Вася бьёт вашу голова» - неприемлемый вариант.

PhantornRU committed:
Apply suggestions from code review

Co-authored-by: Pavel Lysenko <code@ohar.name>

PhantornRU committed:
Перевод расовых действий 2

- Скелет теперь с 25% шансом умеет шутить после распития молока
- Гендеризация
- Перевод действий голема, типа големов
- Расширение имен големов
- Перевод взмахов унатха
- Перевод действий грея
- Перевод взрыва нуклеации

PhantornRU committed:
Изменение имен стеклянного голема

("Линза", "Призма", "Бусинка", "Жемчужина", "Стекловолокно", "Оптика", "Преломлятор", "Преломление", "Отражение")

PhantornRU committed:
Перевод имен големов

PhantornRU committed:
Revert "Перевод имен големов"

This reverts commit b513b18437fba2a8b5ab01809bdfe15d509f2697.

PhantornRU committed:
Revert "Изменение имен стеклянного голема"

This reverts commit d11d598f0316ded90ad3f5336d4a6da578145422.

PhantornRU committed:
небольшой фикс пересечения с пром киданов

PhantornRU committed:
Откат переводов имен големов

Откатил переводы имен големов т.к. слишком много склонений.

TrashDoxx commented:
Пока не мержить. Отсматриваю перевод.

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -130,9 +130,9 @@
-	info_text = "As an <span class='danger'>Adamantine Golem</span>, you possess special vocal cords allowing you to \"resonate\" messages to all golems."
-	prefix = "Adamantine"
-	special_names = null
+	info_text = "Как <span class='danger'>Адамантиновый Голем</span>, вы обладаете особенным голосовыми связками, позволяющие вас \"резонировать\" послания всем големам."
```
```suggestion
	info_text = "Как <span class='danger'>Адамантиновый Голем</span>, вы обладаете особенным голосовыми связками, позволяющие вам «резонировать» послания всем големам."
```

TrashDoxx replied:
```suggestion
	info_text = "Будучи <span class='danger'>адамантиновым големом</span>, вы обладаете особыми голосовыми связками, позволяющими вас \"резонировать\" послания всем големам."
```

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -142,21 +142,21 @@
 	heat_level_2 = 400
 	heat_level_3 = 460
-	info_text = "As a <span class='danger'>Plasma Golem</span>, you burn easily. Be careful, if you get hot enough while burning, you'll blow up!"
+	info_text = "Как <span class='danger'>Плазма Голем</span>, вы легко сгораете. Будьте отсторожны, если вы сильно нагреетесь - вы взорветесь!"
```
Прилагательное, не существительное.
Дублирование «вы» излишне.
Тире вместо дефиса.
```suggestion
	info_text = "Как <span class='danger'>Плазменный голем</span>, вы легко сгораете. Будьте отсторожны, если вы сильно нагреетесь &mdash; взорвётесь!"
```

PhantornRU replied:
Объединил оба варианта

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -206,9 +206,10 @@
 	skinned_type = /obj/item/stack/ore/diamond
-	info_text = "As a <span class='danger'>Diamond Golem</span>, you are more resistant than the average golem."
-	prefix = "Diamond"
+	info_text = "Как <span class='danger'>Алмазный Голем</span>, вы более устойчивы, чем обычный голем."
```
Со строчных букв, пожалуйста.
И старайся использовать причастия, вместо зависимых предложений. Они ёмче, чище и короче.
Аналогично прочие типы големов
```suggestion
	info_text = "Будучи <span class='danger'>алмазным големом</span>, вы прочнее обычных големом."
```

TrashDoxx replied:
```suggestion
	info_text = "Будучи <span class='danger'>алмазным големом</span>, вы прочнее обычных големов."
```

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -472,8 +473,8 @@
-			H.visible_message("<span class='danger'>The [P.name] gets reflected by [H]'s glass skin!</span>", \
-			"<span class='userdanger'>The [P.name] gets reflected by [H]'s glass skin!</span>")
+			H.visible_message("<span class='danger'>[P.name] отразился от стеклянной кожи [H]!</span>", \
+			"<span class='userdanger'>The [P.name] отразился от стеклянной кожи [H]!</span>")
```
Отражается

PhantornRU replied:
внес

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -618,8 +619,8 @@
 	skinned_type = /obj/item/stack/ore/bananium
-	info_text = "As a <span class='danger'>Bananium Golem</span>, you are made for pranking. Your body emits natural honks, and punching people will just harmlessly honk them. Your skin also bleeds banana peels when damaged."
-	prefix = "Bananium"
+	info_text = "As a <span class='danger'>Бананиумовый Голем</span>, вы созданы для розыгрышей. Ваше тело издает естественные гудки, и удары по людям издают безвредные гудки. Ваша кожа также истекает бананами, когда она повреждена."
```
Бананий, не бананиум

TrashDoxx replied:
Как раз таки бананиум. Так на вики указано.

Ohar commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -698,16 +699,16 @@
 	golem_colour = rgb(255, 255, 255)
 	skinned_type = /obj/item/stack/ore/tranquillite
-	info_text = "As a <span class='danger'>Tranquillite Golem</span>, you are capable of creating invisible walls, and can regenerate by drinking your bottle of Nothing."
+	info_text = "As a <span class='danger'>Транквиллитовый Голем</span>, вы можете создавать невидимые стены и регенерировать выпивая бутылки с ничем."
```
Транквилит - с одной «т»

TrashDoxx replied:
```suggestion
	info_text = "Будучи <span class='danger'>транквилитовым големом</span>, вы можете создавать невидимые стены и регенерировать, выпивая бутылки с ничем."
```

Ohar commented on code/modules/mob/living/carbon/human/species/grey.dm:
```diff
@@ -60,11 +60,11 @@
 			H.take_organ_damage(5, 10)
 	else
-		to_chat(H, "<span class='warning'>The water stings[volume < 10 ? " you, but isn't concentrated enough to harm you" : null]!</span>")
+		to_chat(H, "<span class='warning'>Вади жжет вас[volume < 10 ? ", но она недостаточно сконцентрирована, чтобы вам навредить" : null]!</span>")
```
Ваши?

TrashDoxx commented on code/modules/mob/living/carbon/human/species/diona.dm:
```diff
@@ -61,9 +61,10 @@
+		"теряет ветви!",
+		"вытаскивает из тайника бутыль с гербицидом и делает большой глоток!",
+		"распадает гештальт на множество нимф!",
+		"разваливается на множество нимф!")
```
```suggestion
		"разваливается на множество нимф!")
```
То ли дублированный перевод, то ли что. В любом случае, первый вариант коряво написан.

TrashDoxx commented on code/modules/mob/living/carbon/human/species/drask.dm:
```diff
@@ -27,10 +27,10 @@
-		"is holding their breath!")
+		"трёт себя до возгорания!",
+		"давит пальцами на свои большие глаза!",
+		"всасывает теплый воздух!",
```
```suggestion
		"втягивает теплый воздух!",
```
Может лучше без отсосов?

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -63,14 +63,14 @@
-		"is crumbling into dust!",
-		"is smashing their body apart!")
+		"рассыпается в прах!",
+		"разбивает его тело на части!")
```
```suggestion
		"разбивает своё тело на части!")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -63,14 +63,14 @@
 
 	var/golem_colour = rgb(170, 170, 170)
-	var/info_text = "As an <span class='danger'>Iron Golem</span>, you don't have any special traits."
+	var/info_text = "Как <span class='danger'>Железный Голем</span>, у вас нет отличительных особенностей."
```
```suggestion
	var/info_text = "Будучи <span class='danger'>железным големом</span>, вы не обладаете отличительными особенностями."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -142,21 +142,21 @@
 	heat_level_2 = 400
 	heat_level_3 = 460
-	info_text = "As a <span class='danger'>Plasma Golem</span>, you burn easily. Be careful, if you get hot enough while burning, you'll blow up!"
+	info_text = "Как <span class='danger'>Плазма Голем</span>, вы легко сгораете. Будьте отсторожны, если вы сильно нагреетесь - вы взорветесь!"
```
```suggestion
	info_text = "Будучи <span class='danger'>плазменным големом</span>, вы уязвимы к огню. Будьте осторожны, если вы сильно нагреетесь — взорвётесь!"
```

PhantornRU replied:
Объединил оба варианта

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -142,21 +142,21 @@
 	if(H.bodytemperature > 750)
 		if(!boom_warning && H.on_fire)
-			to_chat(H, "<span class='userdanger'>You feel like you could blow up at any moment!</span>")
+			to_chat(H, "<span class='userdanger'>Вы чувствуете что можете взорваться в любой момент!</span>")
```
```suggestion
			to_chat(H, "<span class='userdanger'>Вы чувствуете, что можете взорваться в любой момент!</span>")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -182,17 +182,17 @@
+			to_chat(owner, "<span class='notice'>Вы подожгли себя!</span>")
 		else
-			to_chat(owner, "<span class='warning'>You try to ignite yourself, but fail!</span>")
+			to_chat(owner, "<span class='warning'>Вы попытались поджечь себя, но провалились!</span>")
```
```suggestion
			to_chat(owner, "<span class='warning'>Вы попытались поджечь себя, но неудачно!</span>")
```
Провалиться можно разве что под землю, кхм.

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -222,17 +223,17 @@
-	info_text = "As a <span class='danger'>Gold Golem</span>, you are faster but less resistant than the average golem."
-	prefix = "Golden"
-	special_names = list("Boy")
+	info_text = "Как <span class='danger'>Золотой Голем</span>, вы быстрее, но менее устойчивы, чем обычный голем."
```
```suggestion
	info_text = "Будучи <span class='danger'>золотым големом</span>, вы более быстры, но менее прочны, нежели обычный голем."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -222,17 +223,17 @@
 	punchstunthreshold = 9 //60% chance, from 40%
 	skinned_type = /obj/item/stack/ore/silver
-	info_text = "As a <span class='danger'>Silver Golem</span>, your attacks have a higher chance of stunning."
+	info_text = "Как <span class='danger'>Серебрянный Голем</span>, ваши атаки имеют более высокий шанс оглушения."
```
```suggestion
	info_text = "Будучи <span class='danger'>серебряный големом</span>, вы имеете больший шанс оглушения для своих атак."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -246,7 +247,7 @@
 	speed_mod = 4 //pretty fucking slow
 	skinned_type = /obj/item/stack/ore/iron
-	info_text = "As a <span class='danger'>Plasteel Golem</span>, you are slower, but harder to stun, and hit very hard when punching."
+	info_text = "Как <span class='danger'>Пласталиевый Голем</span>, вы медленнее, но вас сложнее оглушить и вы наносите сильные удары кулаком."
```
```suggestion
	info_text = "Будучи <span class='danger'>пласталиевым големом</span>, вы более медлительные, но вас сложнее оглушить и ваши кулаки причиняют серьёзные повреждения."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -260,7 +261,7 @@
 	golem_colour = rgb(255, 255, 255)
 	skinned_type = /obj/item/stack/ore/titanium
-	info_text = "As a <span class='danger'>Titanium Golem</span>, you are resistant to burn damage and immune to ash storms."
+	info_text = "Как <span class='danger'>Титаниевый Голем</span>, вы устойчивы к урону от ожогов и невосприимчивы к пепельным бурям."
```
```suggestion
	info_text = "Будучи <span class='danger'>титановым големом</span>, вы частично устойчивы к урону от ожогов и невосприимчивы к пепельным бурям."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -278,7 +279,7 @@
 	golem_colour = rgb(136, 136, 136)
 	skinned_type = /obj/item/stack/ore/titanium
-	info_text = "As a <span class='danger'>Plastitanium Golem</span>, you are very resistant to burn damage and immune to both ash storms and lava."
+	info_text = "Как <span class='danger'>Пластитаниумный Голем</span>, вы очень устойчивы к ожогам и невосприимчивы к пепельным бурям и лаве."
```
```suggestion
	info_text = "Будучи <span class='danger'>пластитановым големом</span>, вы крайне устойчивы к ожогам и невосприимчивы к пепельным бурям и лаве."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -301,7 +302,7 @@
 	default_language = "Golem Mindlink"
 	speed_mod = 1 //faster
-	info_text = "As an <span class='danger'>Alloy Golem</span>, you are made of advanced alien materials: you are faster and regenerate over time. You are, however, only able to speak telepathically to other alloy golems."
+	info_text = "Как <span class='danger'>Голем из инопланетных сплавов</span>, вы сделаны из передовых инопланетных материалов: вы быстрее и со временем регенерируете. Однако, вы можете разговаривать только с другими големами из сплава."
```
```suggestion
	info_text = "Будучи <span class='danger'>големом из инопланетных сплавов</span>, вы быстрее и со временем регенерируете. Однако, вы можете разговаривать только с големами из того же материала, что и вы."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -341,9 +342,9 @@
 	heat_level_3 = 400
 
-	dietflags = DIET_HERB		// Plants eat...plants?
+	dietflags = DIET_HERB		// Plants eat...plants? -- естественное развитие растений в природе
```
```suggestion
	dietflags = DIET_HERB		// Plants eat...plants?
```
Кыш-кыш со своими комментариями.

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -341,9 +342,9 @@
+	dietflags = DIET_HERB		// Plants eat...plants? -- естественное развитие растений в природе
 
-	info_text = "As a <span class='danger'>Wooden Golem</span>, you have plant-like traits: you take damage from extreme temperatures, can be set on fire, and have lower armor than a normal golem. You regenerate when in the light and wither in the darkness."
+	info_text = "Как <span class='danger'>Деревянный Голем</span>, вы обладаете особенностями растений: вы получаете урон от экстремальных температур, способный вас поджечь, и у вас меньше брони, чем у обычного голема. Вы регенерируете при свете и увадяете во тьме."
```
```suggestion
	info_text = "Будучи <span class='danger'>деревянным големом</span>, вы обладаете некоторыми особенностями растений: вы получаете урон от экстремальных температур, вас можно поджечь и у вас меньше брони, чем у обычного голема. Вы регенерируете на свету и увядаете во тьме."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -384,7 +385,7 @@
 	golem_colour = rgb(119, 255, 0)
 	skinned_type = /obj/item/stack/ore/uranium
-	info_text = "As an <span class='danger'>Uranium Golem</span>, you emit radiation. It won't harm fellow golems, but organic lifeforms will be affected."
+	info_text = "Как <span class='danger'>Урановый Голем</span>, вы излучаете радиацию. Это не вредит другим големам, но влияет на органические формы жизни."
```
```suggestion
	info_text = "Будучи <span class='danger'>урановым големом</span>, вы излучаете радиацию. Это не вредит другим големам, но влияет на органические формы жизни."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -406,7 +407,7 @@
 	golem_colour = rgb(255, 255, 255)
 	skinned_type = /obj/item/stack/sheet/plastic
-	info_text = "As a <span class='danger'>Plastic Golem</span>, you are capable of ventcrawling if you're naked."
+	info_text = "Как <span class='danger'>Пластиковый Голем</span>, вы способны ползать по вентиляции, если вы раздеты."
```
```suggestion
	info_text = "Будучи <span class='danger'>пластиковым големом</span>, вы способны ползать по вентиляции, если вы раздеты."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -419,13 +420,13 @@
 	brain_mod = 1
 	stamina_mod = 1
-	info_text = "As a <span class='danger'>Sand Golem</span>, you are immune to physical bullets and take very little brute damage, but are extremely vulnerable to burn damage and energy weapons. You will also turn to sand when dying, preventing any form of recovery."
+	info_text = "Как <span class='danger'>Песчаный Голем</span>, вы невосприимчивы к физическим пулям и получаете очень мало грубого урона, но чрезвычайно уязвимы к урону от горения и энергетического оружя. Вы также превратитесь в песок после смерти, что предотвратит любую форму восстановления."
```
```suggestion
	info_text = "Будучи <span class='danger'>песчаным големом</span>, вы невосприимчивы к физическим боеприпасам и получаете очень мало грубого урона. Однако вы чрезвычайно уязвимы к лучам лазерного и энергетического оружия, а также к ожогам. К тому же, вы превратитесь в песок после смерти, что предотвратит любую форму восстановления."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -436,8 +437,8 @@
-			H.visible_message("<span class='danger'>The [P.name] sinks harmlessly in [H]'s sandy body!</span>", \
-			"<span class='userdanger'>The [P.name] sinks harmlessly in [H]'s sandy body!</span>")
+			H.visible_message("<span class='danger'>[P.name] безвредно тонет в песчаном теле [H]!</span>", \
+			"<span class='userdanger'>[P.name] безвредно тонет в песчаном теле  [H]!</span>")
```
```suggestion
			H.visible_message("<span class='danger'>[P.name] тонет в песчаном теле [H] без видимого вреда здоровью!</span>", \
			"<span class='userdanger'>[P.name] тонет в песчаном теле [H] без видимого вреда здоровью!</span>")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -455,14 +456,14 @@
 	brain_mod = 1
 	stamina_mod = 1
-	info_text = "As a <span class='danger'>Glass Golem</span>, you reflect lasers and energy weapons, and are very resistant to burn damage. However, you are extremely vulnerable to brute damage. On death, you'll shatter beyond any hope of recovery."
+	info_text = "Как <span class='danger'>Стеклянный Голем</span>, вы отражаете лазеры и энергетическое оружие и очень устойчивы к урону от горения. Однако вы чрезвычайно уязвимы к грубому урону. После смерти вы разобьется без всякой надежды на восстановление."
```
```suggestion
	info_text = "Будучи <span class='danger'>стеклянным големом</span>, вы отражаете лучи лазерного и энергетического оружия, а также крайне устойчивы к урону от ожогов. Однако вы чрезвычайно уязвимы к грубому урону и физическим боеприпасам. К тому же, после смерти вы разобьётесь без всякой надежды на восстановление."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -455,14 +456,14 @@
 /datum/species/golem/glass/handle_death(gibbed, mob/living/carbon/human/H)
 	playsound(H, "shatter", 70, 1)
-	H.visible_message("<span class='danger'>[H] shatters!</span>")
+	H.visible_message("<span class='danger'>[H] разбился в дребезги!</span>")
```
```suggestion
	H.visible_message("<span class='danger'>[H] разбился вдребезги!</span>")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -488,7 +489,7 @@
 	golem_colour = rgb(51, 51, 255)
 	skinned_type = /obj/item/stack/ore/bluespace_crystal
-	info_text = "As a <span class='danger'>Bluespace Golem</span>, you are spatially unstable: You will teleport when hit, and you can teleport manually at a long distance."
+	info_text = "Как <span class='danger'>Блюспейс Голем</span>, вы пространственно нестабильны: Вы будете телепортироваться при ударе, и вы можете телепортироваться вручную на большое расстояние.."
```
```suggestion
	info_text = "Будучи <span class='danger'>блюспейс-големом</span>, вы пространственно нестабильны: вы будете телепортироваться при получении ударов. Также вы можете телепортироваться вручную на большое расстояние."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -575,13 +576,13 @@
 	activated = TRUE
 	var/mob/living/carbon/human/H = owner
-	H.visible_message("<span class='warning'>[H] starts vibrating!</span>", "<span class='danger'>You start charging your bluespace core...</span>")
+	H.visible_message("<span class='warning'>[H] начинает вибрировать!</span>", "<span class='danger'>Вы начали заряжать свое блюспейс ядро...</span>")
```
```suggestion
	H.visible_message("<span class='warning'>[H] начинает вибрировать!</span>", "<span class='danger'>Вы начали заряжать своё блюспейс-ядро...</span>")
```

TrashDoxx reviewed (changes requested):
Частично вычитал перевод. Из общих замечаний: переведены не все префиксы, а также не все имена для големов. Следует дополнить перевод. Остальное вычитаю завтра, сил и мотивации уже нет на это. Лучше бы ты ещё более точечно дробил переводы.

PhantornRU committed:
Update grey.dm

PhantornRU committed:
Update machine.dm

PhantornRU committed:
доперевод воксов

PhantornRU committed:
Apply suggestions - Убран отсос драсков

Драски больше не сосут :(

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

PhantornRU committed:
Apply suggestions from code review

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

PhantornRU committed:
Перевод конечностей слайма + возврат имен големов

PhantornRU commented:
Големов откатил. Я их изначально тут откатывал, но просто не убрал некоторых по ошибке.
Големы будут отдельным ПРом.
Перевел отращивание конечностей слаймов, поправил пересекающиеся коммиты
Драски больше не отсасывают
Воксы, машины  и греи теперь еще больше русоязычные, ведь убрана косоязычность по саггестам

PhantornRU committed:
Фикс перевода "резонирования голема"

PhantornRU commented:
Одобрено Авоутом
https://discord.com/channels/617003227182792704/755125334097133628/978973948018565161

TrashDoxx commented on code/modules/mob/living/carbon/human/species/vulpkanin.dm:
```diff
@@ -57,11 +57,11 @@
+		"выдавливает когтями свои глазницы!",
+		"сворачивает себе шею!",
+		"задерживает дыхание!")
+		
```
```suggestion

```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/vox.dm:
```diff
@@ -121,7 +121,7 @@
 			to_chat(H, "<span class='boldannounce'>Could not find an empty slot for internals! Please report this as a bug</span>")
 	H.internal = internal_tank
-	to_chat(H, "<span class='notice'>You are now running on nitrogen internals from the [internal_tank]. Your species finds oxygen toxic, so you must breathe nitrogen only.</span>")
+	to_chat(H, "<span class='notice'>Теперь вы живете на азоте из [internal_tank]. Ваш вид считает кислород токсичным, поэтому вы должны дышать только азотом.</span>")
```
```suggestion
	to_chat(H, "<span class='notice'>Теперь вы живете на азоте из [internal_tank]. Кислород токсичен для вашего вида, поэтому вы должны дышать только азотом.</span>")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/unathi.dm:
```diff
@@ -111,20 +120,20 @@
 					return
 			if(user.getStaminaLoss() >= 60) //Bit higher as you don't need to start, just would need to keep going with the tail lash.
-				to_chat(user, "<span class='warning'>You run out of momentum!</span>")
+				to_chat(user, "<span class='warning'>Вы вышли из инерции!</span>")
```
```suggestion
				to_chat(user, "<span class='warning'>Вы выбились из сил!</span>")
```
Насколько я понял из контекста, речь именно об этом.

PhantornRU replied:
Здесь идет окончание махов хвостами когда ты прекращаешь вращаться по инерции. Но в принципе. Пускай выбивается из сил, коль зависимость идет от стамины

TrashDoxx commented on code/modules/mob/living/carbon/human/species/slime.dm:
```diff
@@ -158,30 +160,43 @@
 
 	if(!missing_limbs.len)
-		to_chat(H, "<span class='warning'>You're not missing any limbs!</span>")
+		to_chat(H, "<span class='warning'>Вы не потеряли ни одной конечности!</span>")
```
```suggestion
		to_chat(H, "<span class='warning'>Ваши конечности на месте!</span>")
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -618,8 +618,8 @@
 	skinned_type = /obj/item/stack/ore/bananium
-	info_text = "As a <span class='danger'>Bananium Golem</span>, you are made for pranking. Your body emits natural honks, and punching people will just harmlessly honk them. Your skin also bleeds banana peels when damaged."
-	prefix = "Bananium"
+	info_text = "As a <span class='danger'>Бананиумовый Голем</span>, вы созданы для розыгрышей. Ваше тело издает естественные гудки, и удары по людям издают безвредные гудки. Ваша кожа также истекает бананами, когда она повреждена."
```
```suggestion
	info_text = "Будучи <span class='danger'>бананиумовым голем</span>, вы созданы для розыгрышей. Ваше тело издает естественные гудки, и удары по людям издают безвредные гудки. Ваша кожа также истекает бананами, когда она повреждена."
```

TrashDoxx commented on code/modules/mob/living/carbon/human/species/golem.dm:
```diff
@@ -222,17 +222,17 @@
-	prefix = "Golden"
-	special_names = list("Boy")
+	info_text = "Будучи <span class='danger'>золотым големом</span>, вы более быстры, но менее прочны, нежели обычный голем."
+	prefix = "Gold"
```
Зачем изменён префикс?

PhantornRU replied:
Временный возврат к големам. Их изменения в ПРе с именами

TrashDoxx reviewed (changes requested):
Бегло пробежался, лень трудиться.

PhantornRU committed:
Apply suggestions from code review

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

PhantornRU committed:
Update code/modules/mob/living/carbon/human/species/golem.dm

Co-authored-by: TrashDoxx <67621641+TrashDoxx@users.noreply.github.com>

Bizzonium commented:
![image](https://user-images.githubusercontent.com/8555356/170880408-faec3a8e-5f62-4c99-8b8e-4c4c0a36092d.png)

PhantornRU committed:
Merge branch 'master220' into race_s_translate

PhantornRU committed:
Перевод рас - решение конфликтов

PhantornRU committed:
Расовый перевод - решение конфликта диет

PhantornRU commented:
> ![image](https://user-images.githubusercontent.com/8555356/170880408-faec3a8e-5f62-4c99-8b8e-4c4c0a36092d.png)

Конфликт решил

PhantornRU committed:
Големы - решен конфликт с диетами
